app/feature/feature-profile/src/test/kotlin/com/hedvig/android/feature/profile/myinfo/MyInfoPresenterTest.kt

```diff
@@ -30,6 +31,7 @@ class MyInfoPresenterTest {
     "1234567",
   )
 
+  @Ignore("https://kotlinlang.slack.com/archives/C5UPMM0A0/p1731449652340769")
```

Comment:
What is there in the slack?:) I don't have access to this

Reply:
Yes you are right, this is in kotlinlang slack channel, not in ours. I would've put the link to the archives https://slack-chats.kotlinlang.org/c/arrow but that does not yet have my new message synced. Once it is I will try to update it.
It's basically just me explaining what exception I get there and asking for help, since the exception is something that I have absolutely no clue how to fix. Here is the exception for referece:
The interesting (I think) part:
```
Caused by: java.lang.IllegalAccessError: failed to access class arrow.core.raise.RaiseKt__RaiseAccumulateKt from class com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt (arrow.core.raise.RaiseKt__RaiseAccumulateKt and com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt are in unnamed module of loader 'app')
```
The entire message
```
Expected item for molecule presenter turbine but found Error(IllegalAccessError)
app.cash.turbine.TurbineAssertionError: Expected item for molecule presenter turbine but found Error(IllegalAccessError)
	at app//app.cash.turbine.TurbineAssertionError$Companion.invoke(TurbineAssertionError.kt:32)
	at app//app.cash.turbine.ChannelKt.unexpectedEvent(channel.kt:253)
	at app//app.cash.turbine.ChannelKt.awaitItem(channel.kt:192)
	at app//app.cash.turbine.ChannelTurbine$awaitItem$2.invokeSuspend(Turbine.kt:210)
	at app//app.cash.turbine.ChannelTurbine$awaitItem$2.invoke(Turbine.kt)
	at app//app.cash.turbine.ChannelTurbine$awaitItem$2.invoke(Turbine.kt)
	at app//app.cash.turbine.ChannelTurbine.withTurbineTimeout(Turbine.kt:113)
	at app//app.cash.turbine.ChannelTurbine.awaitItem(Turbine.kt:210)
	at app//app.cash.turbine.TurbineTestContextImpl.awaitItem(flow.kt)
	at app//com.hedvig.android.molecule.test.MoleculePresenterTestContextImpl.awaitItem(MoleculeTest.kt:52)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1$1.invokeSuspend(MyInfoPresenterTest.kt:55)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1$1.invoke(MyInfoPresenterTest.kt)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1$1.invoke(MyInfoPresenterTest.kt)
	at app//com.hedvig.android.molecule.test.MoleculeTestKt$test$3.invokeSuspend(MoleculeTest.kt:23)
	at app//com.hedvig.android.molecule.test.MoleculeTestKt$test$3.invoke(MoleculeTest.kt)
	at app//com.hedvig.android.molecule.test.MoleculeTestKt$test$3.invoke(MoleculeTest.kt)
	at app//app.cash.turbine.FlowKt$test$2.invokeSuspend(flow.kt:143)
	at app//app.cash.turbine.FlowKt$test$2.invoke(flow.kt)
	at app//app.cash.turbine.FlowKt$test$2.invoke(flow.kt)
	at app//app.cash.turbine.FlowKt$turbineScope$2$1.invokeSuspend(flow.kt:91)
	at app//app.cash.turbine.FlowKt$turbineScope$2$1.invoke(flow.kt)
	at app//app.cash.turbine.FlowKt$turbineScope$2$1.invoke(flow.kt)
	at app//kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:42)
	at app//kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:261)
	at app//app.cash.turbine.FlowKt$turbineScope$2$scopeFn$1.invokeSuspend(flow.kt:83)
	at app//app.cash.turbine.FlowKt$turbineScope$2$scopeFn$1.invoke(flow.kt)
	at app//app.cash.turbine.FlowKt$turbineScope$2$scopeFn$1.invoke(flow.kt)
	at app//app.cash.turbine.FlowKt$turbineScope$2.invokeSuspend(flow.kt:88)
	at app//app.cash.turbine.FlowKt$turbineScope$2.invoke(flow.kt)
	at app//app.cash.turbine.FlowKt$turbineScope$2.invoke(flow.kt)
	at app//app.cash.turbine.CoroutinesKt$reportTurbines$2.invokeSuspend(coroutines.kt:89)
	at app//app.cash.turbine.CoroutinesKt$reportTurbines$2.invoke(coroutines.kt)
	at app//app.cash.turbine.CoroutinesKt$reportTurbines$2.invoke(coroutines.kt)
	at app//kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:42)
	at app//kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:164)
	at app//kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at app//app.cash.turbine.CoroutinesKt.reportTurbines(coroutines.kt:88)
	at app//app.cash.turbine.FlowKt.turbineScope-k1IrOU0(flow.kt:80)
	at app//app.cash.turbine.FlowKt.test-C2H2yOE(flow.kt:141)
	at app//app.cash.turbine.FlowKt.test-C2H2yOE$default(flow.kt:136)
	at app//com.hedvig.android.molecule.test.MoleculeTestKt.test(MoleculeTest.kt:22)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1.invokeSuspend(MyInfoPresenterTest.kt:38)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1.invoke(MyInfoPresenterTest.kt)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1.invoke(MyInfoPresenterTest.kt)
	at app//kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt$runTest$2$1$1.invokeSuspend(TestBuilders.kt:318)
	at app//kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at app//kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at app//kotlinx.coroutines.test.TestDispatcher.processEvent$kotlinx_coroutines_test(TestDispatcher.kt:24)
	at app//kotlinx.coroutines.test.TestCoroutineScheduler.tryRunNextTaskUnless$kotlinx_coroutines_test(TestCoroutineScheduler.kt:99)
	at app//kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt$runTest$2$1$workRunner$1.invokeSuspend(TestBuilders.kt:327)
	at app//kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at app//kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at app//kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
	at app//kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
	at app//kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
	at app//kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at app//kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
	at app//kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at app//kotlinx.coroutines.test.TestBuildersJvmKt.createTestResult(TestBuildersJvm.kt:10)
	at app//kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt.runTest-8Mi8wO0(TestBuilders.kt:310)
	at app//kotlinx.coroutines.test.TestBuildersKt.runTest-8Mi8wO0(Unknown Source)
	at app//kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt.runTest-8Mi8wO0(TestBuilders.kt:168)
	at app//kotlinx.coroutines.test.TestBuildersKt.runTest-8Mi8wO0(Unknown Source)
	at app//kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt.runTest-8Mi8wO0$default(TestBuilders.kt:160)
	at app//kotlinx.coroutines.test.TestBuildersKt.runTest-8Mi8wO0$default(Unknown Source)
	at app//com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest.if phone and email are updated they are received in state(MyInfoPresenterTest.kt:34)
	at java.base@21.0.1/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base@21.0.1/java.lang.reflect.Method.invoke(Method.java:580)
	at app//org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at app//org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at app//org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at app//org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at app//org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at app//org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at app//org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at app//org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at app//org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at app//org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at app//org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at app//org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at app//org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at app//org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at app//org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at app//org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at app//org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:112)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:40)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:54)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:53)
	at java.base@21.0.1/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base@21.0.1/java.lang.reflect.Method.invoke(Method.java:580)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:92)
	at jdk.proxy1/jdk.proxy1.$Proxy4.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:183)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:132)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:103)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:63)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:121)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:71)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
Caused by: java.lang.IllegalAccessError: failed to access class arrow.core.raise.RaiseKt__RaiseAccumulateKt from class com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt (arrow.core.raise.RaiseKt__RaiseAccumulateKt and com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt are in unnamed module of loader 'app')
	at com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt.validate(MyInfoViewModel.kt:411)
	at com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt.access$validate(MyInfoViewModel.kt:1)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenter$present$3$1.invokeSuspend(MyInfoViewModel.kt:108)
	at _COROUTINE._BOUNDARY._(CoroutineDebugging.kt:42)
	at app.cash.molecule.MoleculeKt$immediateClockFlow$1.invokeSuspend(molecule.kt:61)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:226)
	at app.cash.turbine.FlowKt$collectTurbineIn$job$1.invokeSuspend(flow.kt:206)
Caused by: java.lang.IllegalAccessError: failed to access class arrow.core.raise.RaiseKt__RaiseAccumulateKt from class com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt (arrow.core.raise.RaiseKt__RaiseAccumulateKt and com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt are in unnamed module of loader 'app')
	at com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt.validate(MyInfoViewModel.kt:411)
	at com.hedvig.android.feature.profile.myinfo.MyInfoViewModelKt.access$validate(MyInfoViewModel.kt:1)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenter$present$3$1.invokeSuspend(MyInfoViewModel.kt:108)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.EventLoop.processUnconfinedEvent(EventLoop.common.kt:65)
	at kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:231)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:149)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
	at kotlinx.coroutines.flow.SharedFlowImpl.tryEmit(SharedFlow.kt:414)
	at com.hedvig.android.molecule.test.MoleculePresenterTestContextImpl.sendEvent(MoleculeTest.kt:42)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1$1.invokeSuspend(MyInfoPresenterTest.kt:54)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1$1.invoke(MyInfoPresenterTest.kt)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1$1.invoke(MyInfoPresenterTest.kt)
	at com.hedvig.android.molecule.test.MoleculeTestKt$test$3.invokeSuspend(MoleculeTest.kt:23)
	at com.hedvig.android.molecule.test.MoleculeTestKt$test$3.invoke(MoleculeTest.kt)
	at com.hedvig.android.molecule.test.MoleculeTestKt$test$3.invoke(MoleculeTest.kt)
	at app.cash.turbine.FlowKt$test$2.invokeSuspend(flow.kt:143)
	at app.cash.turbine.FlowKt$test$2.invoke(flow.kt)
	at app.cash.turbine.FlowKt$test$2.invoke(flow.kt)
	at app.cash.turbine.FlowKt$turbineScope$2$1.invokeSuspend(flow.kt:91)
	at app.cash.turbine.FlowKt$turbineScope$2$1.invoke(flow.kt)
	at app.cash.turbine.FlowKt$turbineScope$2$1.invoke(flow.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:42)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:261)
	at app.cash.turbine.FlowKt$turbineScope$2$scopeFn$1.invokeSuspend(flow.kt:83)
	at app.cash.turbine.FlowKt$turbineScope$2$scopeFn$1.invoke(flow.kt)
	at app.cash.turbine.FlowKt$turbineScope$2$scopeFn$1.invoke(flow.kt)
	at app.cash.turbine.FlowKt$turbineScope$2.invokeSuspend(flow.kt:88)
	at app.cash.turbine.FlowKt$turbineScope$2.invoke(flow.kt)
	at app.cash.turbine.FlowKt$turbineScope$2.invoke(flow.kt)
	at app.cash.turbine.CoroutinesKt$reportTurbines$2.invokeSuspend(coroutines.kt:89)
	at app.cash.turbine.CoroutinesKt$reportTurbines$2.invoke(coroutines.kt)
	at app.cash.turbine.CoroutinesKt$reportTurbines$2.invoke(coroutines.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:42)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:164)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at app.cash.turbine.CoroutinesKt.reportTurbines(coroutines.kt:88)
	at app.cash.turbine.FlowKt.turbineScope-k1IrOU0(flow.kt:80)
	at app.cash.turbine.FlowKt.test-C2H2yOE(flow.kt:141)
	at app.cash.turbine.FlowKt.test-C2H2yOE$default(flow.kt:136)
	at com.hedvig.android.molecule.test.MoleculeTestKt.test(MoleculeTest.kt:22)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1.invokeSuspend(MyInfoPresenterTest.kt:38)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1.invoke(MyInfoPresenterTest.kt)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest$if phone and email are updated they are received in state$1.invoke(MyInfoPresenterTest.kt)
	at kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt$runTest$2$1$1.invokeSuspend(TestBuilders.kt:318)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.test.TestDispatcher.processEvent$kotlinx_coroutines_test(TestDispatcher.kt:24)
	at kotlinx.coroutines.test.TestCoroutineScheduler.tryRunNextTaskUnless$kotlinx_coroutines_test(TestCoroutineScheduler.kt:99)
	at kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt$runTest$2$1$workRunner$1.invokeSuspend(TestBuilders.kt:327)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:95)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:69)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:47)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at kotlinx.coroutines.test.TestBuildersJvmKt.createTestResult(TestBuildersJvm.kt:10)
	at kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt.runTest-8Mi8wO0(TestBuilders.kt:310)
	at kotlinx.coroutines.test.TestBuildersKt.runTest-8Mi8wO0(Unknown Source)
	at kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt.runTest-8Mi8wO0(TestBuilders.kt:168)
	at kotlinx.coroutines.test.TestBuildersKt.runTest-8Mi8wO0(Unknown Source)
	at kotlinx.coroutines.test.TestBuildersKt__TestBuildersKt.runTest-8Mi8wO0$default(TestBuilders.kt:160)
	at kotlinx.coroutines.test.TestBuildersKt.runTest-8Mi8wO0$default(Unknown Source)
	at com.hedvig.android.feature.profile.myinfo.MyInfoPresenterTest.if phone and email are updated they are received in state(MyInfoPresenterTest.kt:34)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:112)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:40)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:54)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:53)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:92)
	at jdk.proxy1/jdk.proxy1.$Proxy4.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:183)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:132)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:103)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:63)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:121)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:71)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
```